functions/Invoke-DbaDiagnosticQuery.ps1

```diff
@@ -305,9 +398,28 @@ Then it will export the results to Export-DbaDiagnosticQuery.
                                 Result           = $result | Select-Object * -ExcludeProperty RowError, RowState, Table, ItemArray, HasErrors
```

Comment:
use select-defaultview

Reply:
Also should be done as `Select-DefaultView -InputObject $result...` instead of piping it for better performance.